src/lib/DriverError.ml

```diff
@@ -0,0 +1,2 @@
+
+type error_with_maybe_span =  ErrorWithMaybeSpan of DriverMessage.message * Basis.LexingUtil.span option
```

Comment:
I think we can make this a little more idiomatic, like:

```ocaml
type error = {message : DriverMessage.message; span : Basis.LexingUtil.span option}
```

Reply:
I changed this up to be better by removing DriverError entirely and making a more clear DriverMessage module